Communication.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Communication.cpp
+ *
+ *  Created on: 8 June 2020
+ *      Author: Zhuoheng Li
+ */
+
+#define COMMUNICATION_DEBUG
+
+#include "Communication.h"
+//TODO: #include "OBCDataContainer.h"

```

Comment:
I guess this can be done now that OBCdatacontainer is in the branch?

Reply:
I agree!
OBCdatacontainer has not been used in Communication.cpp yet.
However, it will be used if we add the functions to collect command from COMMS. We don't add such functions because the interface is still unknown to us at this moment.